### PR TITLE
Fixes and adds test for name_table bug

### PR DESCRIPTION
**Pull request by rdaly525**

### CODE REVIEW

**Review by cdonovick — APPROVED** [review]

**Comment by cdonovick** [comment]
Approved pending passing CI modulo outstanding floating point issues

File: tests/test_smt_utils.py [hunk 1]
```diff
@@ -23,28 +29,27 @@ def test_fc():
...
-|   |   |   ((! y_0) & z_0),
-|   |   |   ((! z_0) & x_0)
+|   |   |   (! y_YI),
+|   |   |   (! z_ZI)
 |   |   )
 |   ),
 |   True
-)'''
-
+)'''.replace("XI", XI).replace("YI", YI).replace("ZI", ZI)
```
Comment (cdonovick) [inline tests/test_smt_utils.py:39]: could use a format string.

**Review by cdonovick — COMMENTED** [review]
Not the most beautiful tests but Ill accept it.

**Comment by rdaly525** [comment]
@leonardt @cdonovick, should I merge?

**Comment by leonardt** [comment]
Ok with me since it reduces the number of failing tests.

**Pull request merged**